solace-spring-boot-autoconfigure/solace-java-spring-boot-autoconfigure/src/test/java/com/solace/spring/boot/autoconfigure/SolaceOAuthClientConfigurationTest.java

```diff
@@ -21,8 +21,9 @@ public static class TestApp {
   void verifyApplicationContextContainsRequiredBeans() {
     try (ConfigurableApplicationContext context = new SpringApplicationBuilder()
         .profiles("oauthConfigIT").sources(TestApp.class)
-        .properties(String.format("%s=%s", "solace.java.api-properties.AUTHENTICATION_SCHEME",
-            "AUTHENTICATION_SCHEME_OAUTH2"))
+        .properties(
+            String.format("%s=%s", "solace.java.api-properties.AUTHENTICATION_SCHEME", "AUTHENTICATION_SCHEME_OAUTH2"),
+            String.format("%s=%s", "solace.java.oauth2ClientRegistrationId", "my-oauth2-client"))
```

Comment:
might want to fix the formatting here to be consistent with previous property.

So since `"solace.java.api-properties.AUTHENTICATION_SCHEME"` is `kebab-case`, then `oauth2ClientRegistrationId` should be `oauth2-client-registration-id`, etc.

Reply:
This would also give us a wee bit of coverage that `oauth2ClientRegistrationId` works in a different formats than camel-case (which I see is what you used elsewhere in this test class).